### ISSUE TITLE
Move popup into separate component

### ISSUE THREAD

**Issue by anselanza** [RandomStudio/large-language-trees#4]

**Comment by anselanza** [comment]
Make a component that can render the details for a single plant. NOT a route/page.

Component will look something like this:
```
{#if selectedPlant}
<PlantPopup plant={selectedPlant} />
{/if}
```

Implement logic so that popup for that plant shows when clicking it.

Logic for deleting.

**Comment by hectoralbiges** [comment]
Problem solved #8
